### DISCUSSION TITLE
Getting parent value on selection

### DISCUSSION THREAD

**Discussion by alprbykn** [dowjones/react-dropdown-tree-select#536] [Q&A]
Is it possible to get value or label of selected node's parent. Not like ui-id such as  "rdts10-0-0", I want it's value as i.e U.K.
I also wonder how can access the selectednodes from outside.

**Comment by mrchief** [comment]
You get the id of the parent so you'd need to traverse but you can also pass in an extra property (or multiple) that you'd like to see `onChange` https://github.com/dowjones/react-dropdown-tree-select#data

![image](https://user-images.githubusercontent.com/781818/144147742-82546dec-e57a-4daf-8c10-299681d39fab.png)

**Comment by mrchief** [comment]
I created a Codesandbox for this https://codesandbox.io/s/breadcrumbs-with-react-dropdown-tree-select-id6ik?file=/index.js

![image](https://user-images.githubusercontent.com/781818/144267892-4ef9351c-268c-4e5a-a0fa-0549ba0ba828.png)

**Reply by maria88ahmed** [reply]
@mrchief , Is that possible to show the breadcrumbs instead of the selected node's text ?
![DropdownTreeSelect](https://user-images.githubusercontent.com/65946463/144980961-f25a9396-6dc1-4151-a994-4667087d50ef.png)

**Reply by mrchief** [reply]
Not within the component but you can hide the component's tags using CSS and supply your own tags. You'd add remove your own tags by reacting to node change events and using the breadcrumbs prop from the `selectedNode` object you get with those events.

**Reply by maria88ahmed** [reply]
Thanks @mrchief , I did override the tagLabel , with the breadcrumbs value using HOC. Is there any better way of doing it ?

**Reply by mrchief** [reply]
@maria88ahmed the extensibility via CSS overrides and HoC was part of the core design so you're on the right track. It was done to allow for composition to extend and support these extended use-cases without creating code/feature bloat in the component.
